src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -175,6 +210,7 @@ def load(
         profile_name: Optional[str] = None,
         profile_type: Optional[str] = None,
         check_missing_props: bool = True,
+        opt_in: Optional[bool] = False,
```

Comment:
Should this default to True?
I guess we would want ENV to always be loaded, and for the user to opt_out?